pkg/controller/instance/controller.go

```diff
@@ -154,8 +161,9 @@ func (c *Controller) Reconcile(ctx context.Context, req ctrl.Request) error {
 		log:                         log,
 		gvr:                         c.gvr,
 		client:                      executionClient,
+		restMapper:                  c.clientSet.RESTMapper(),
```

Comment:
Im wondering about this. I believe the RESTMapper from the client set may be wrong? im wondering if the correct client for resolving mappings here should be the stuff behind execution client?
will have to double check this again!

Reply:
Should we revert to the old state where we dont include restMapper as part of clientset ?